src/Apps/CollectorProfile/Routes/Saves2/Components/Actions/DeleteSavesModal.tsx

```diff
@@ -85,10 +85,7 @@ export const DeleteSavesModal: React.FC<Props> = ({ collection, onClose }) => {
           </Button>
         </Flex>
       }
-      width={[
-        "100%",
-        753, // vs. 713px for create and edit modals and 673px for manage-artworks modal -- should be consistent (700?) or maybe just "auto"?
-      ]}
+      width={["100%", 740]}
```

Comment:
`740px` is used here because the horizontal margins are taken into account (`20px` on the left and `20px` on the right)

![screenshot](https://user-images.githubusercontent.com/3513494/224746705-980a8621-c7bb-4b31-9d19-969a4b34d649.png)